installer/roles/provision-mobile-catalog/tasks/main.yml

```diff
@@ -0,0 +1,82 @@
+---
+
+# tasks for provision mobile catalog on an existing openshift cluster
+# NOTE: Ansible Service Broker should be already provisoned in this cluster
+- name: Get current user
+  shell: "oc whoami"
+  register: "oc_whoami_output"
+
+- set_fact:
+    current_user: "{{oc_whoami_output.stdout}}"
+
+- name: Login as system admin
```

Comment:
Not sure we should assume this will work. I think we should allow the user to give us a user while still keeping sensible defaults